src/modules/string/index.ts

```diff
@@ -241,6 +241,93 @@ export class StringModule {
     ).join('');
   }
 
+  /**
+   * Returns a [binary](https://en.wikipedia.org/wiki/Binary_number) string.
+   *
+   * @param options The optional options object.
+   * @param options.length The number or range of characters to generate after the prefix. Defaults to `1`.
+   * @param options.prefix Prefix for the generated number. Defaults to `'0b'`.
+   *
+   * @example
+   * faker.string.binary() // '0b1'
+   * faker.string.binary({ length: 10 }) // '0b1101011011'
+   * faker.string.binary({ length: { min: 5, max: 10 } }) // '0b11101011'
+   * faker.string.binary({ prefix: '0b' }) // '0b1'
+   * faker.string.binary({ length: 10, prefix: '#' }) // '#1101011011'
```

Comment:
I think this example doesn't make much sense, but we could swap it out with `bin` for binary 🤔 

```suggestion
   * faker.string.binary({ length: 10, prefix: 'bin' }) // 'bin1101011011'
```

Please let have @ST-DDT also have a look over my suggestion before applying it as I'm not sure

---

Edit: Oh, below in the tests you use already `bin_` and `oct_`!
Please use these here in the examples instead

Reply:
Sure, sounds good!